docs/cmake-project-view.md

```diff
@@ -0,0 +1,116 @@
+            "projectStatusVisibility": "visible", "hidden" 
+        }, 
+        "configure": { 
+            	"projectStatusVisibility": "visible", "hidden" 
+        }, 
+        "configurePreset": { 
+            	"statusBarVisibility": "visible", "icon", "compact", "hidden" 
+            	"statusBarLength": 20
+        }, 
+        "kit": { 
+            	"statusBarVisibility": "visible", "icon", "compact", "hidden" 
+            	"statusBarLength": 20 
+        }, 
+        "variantStatus": { 
+            	"statusBarVisibility": "visible", "icon", "compact", "hidden" 
+        }, 
+        "build": { 
+            	"statusBarVisibility": "visible", "icon", "hidden" 
+            	"projectStatusVisibility": "visible", "hidden" 
+        }, 
+        "buildPreset": { 
+            	"statusBarVisibility": "visible", "icon", "compact", "hidden" 
+            	"statusBarLength": 20 
+        }, 
+        "buildTarget": { 
+            	"statusBarVisibility": "visible", "compact", "hidden" 
+            	"statusBarLength": 20 
+        }, 
+        "ctest": { 
+            	"statusBarVisibility": "visible", "icon", "compact", "hidden" 
+            	"statusBarLength": 20 
+            	"color": true, false 
+            	"projectStatusVisibility": "visible", "hidden" 
+        }, 
+        "testPreset": { 
+            	"statusBarVisibility": "visible", "icon", "compact", "hidden" 
+            	"statusBarLength": 20
+        },
+        "launchTarget": { 
+            	"statusBarVisibility": "visible", "compact", "hidden" 
+            	"statusBarLength": 20 
+        }, 
+        "debug": { 
+            	"statusBarVisibility": "visible", "hidden" 
+            	"projectStatusVisibility": "visible", "hidden" 
+        },
+        "launch": { 
+            	"statusBarVisibility": "visible", "hidden" 
+            	"projectStatusVisibility": "visible", "hidden" 
+        }
+}
+```
+
+* `folder` is the active folder in your workspace. This is where the project is scoped to.
+* `configure` is associated with the CMake Configure action. It can only be found in the Project Status View and can't be seen in the status bar, hence no `statusBarVisibility` property value
+* `configurePreset` is associated with the CMake Configure Preset. When selected, the user can modify their active CMake Configure Preset from the list detected in their CMakePresets.json and CMakeUserPresets.json (if found in project)
+* `kit` is associated with the active kit selected (when CMake Presets aren't present). When selected, the user can modify their active kit. To learn more about kits, please see [our kit documentation](https://github.com/microsoft/vscode-cmake-tools/docs/kits.md)
+* `variantStatus` is associated with the active variant status (when CMake Presets aren't present). When selected, the user can modify their active variant. To learn more about variants, please see [our variant documentation](https://github.com/microsoft/vscode-cmake-tools/docs/variants.md)
```

Comment:
so this is the active variant and the status. they're two things. also maybe worth mentioning that the status doesn't show on the project status view